src/addrman.h

```diff
@@ -415,7 +415,7 @@ class CAddrMan
             }
         }
         if (nLost + nLostUnk > 0) {
-            LogPrint("addrman", "addrman lost %i new and %i tried addresses due to collisions\n", nLostUnk, nLost);
```

Comment:
We don't put the \n explicitly like Bitcoin, rather it is in the logger itself. Please remove the \n as well. Thanks.